go/ephemeral/errors.go

```diff
@@ -52,15 +54,30 @@ func newEKCorruptedErr(ctx context.Context, g *libkb.GlobalContext, boxType EKTy
 	return newEphemeralKeyError(debugMsg, "")
 }
 
-func newEphemeralKeyError(debugMsg, humanMsg string) EphemeralKeyError {
+func humanMsgWithPrefix(humanMsg string) string {
 	if humanMsg == "" {
-		humanMsg = defaultHumanErr
+		humanMsg = DefaultHumanErrMsg
+	} else if !strings.Contains(humanMsg, DefaultHumanErrMsg) {
```

Comment:
I'm not sure what's going on here but I assume it's good

Reply:
just a sanity check so we don't append the default message twice